_docs-sources/pipelines/security/multi-account.md

```diff
@@ -0,0 +1,54 @@
+# Multiple Infrastructure-Live Repos
```

Comment:
Nit: I personally find it easier to review and maintain the content when it's all on one line versus us manually adding line breaks.

Reply:
sorry, bad habit from Garmin where they very strictly enforced the 80 character line limit even in text files

Reply:
Oh interesting. Yeah, let's standardize on one line then.